web/js/gerbera.tree.js

```diff
@@ -76,8 +77,9 @@ GERBERA.Tree = (function () {
       .fail(GERBERA.App.error)
   }
 
-  var loadTree = function (response) {
+  var loadTree = function (response, config) {
     var tree = $('#tree')
+    config = $.extend({}, treeViewConfig, config)
```

Comment:
Added extend here to make `treeViewConfig` more testable after adding both `onExpand` and `onSelection` to the same click.